cs251tk/toolkit/args.py

```diff
@@ -65,7 +70,7 @@ def get_args():
     return parser
 
 
-def massage_args(args, students):
+def massage_args(args, students, now=datetime.date.today()):
```

Comment:
Good plan.  In general, adding arguments isn't the *best* way to model this, however.  I don't know if the unit testing framework we're using here works for this, but in a lot of my projects I'll use a Test Double to model this—replacing what would be called by `datetime.date.today()` with our own method which returns what we want to depending on our tests.  Since this is an optional argument, though, it doesn't add much clutter.

Either way, this will work, and in a sense this is quite elegant.